### PR TITLE
Fix/db configuration

### PR DESCRIPTION
**Pull request by outcatcher**
Fixed not working postgresql integration for too_simple_server v.0.2.3

### CODE REVIEW

**Comment by outcatcher** [comment]
Not ready yet

**Comment by outcatcher** [comment]
Incorrectly closed

File: too_simple_server/database.py [hunk 1]
```diff
@@ -19,21 +19,39 @@ class Entity(BaseModel):
...
 
 
-def init_db():
-    configuration = load_configuration()
+def _create_psql_database(db_name, user, password, host, port):
+    from psycopg2 import connect
+    from psycopg2.errors import DuplicateDatabase
+    from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+    conn = connect(dbname="postgres", user=user, password=password, host=host, port=port)
```
Comment (anton-sidelnikov) [inline too_simple_server/database.py:27]: db_name variable not used?

Comment (outcatcher) [reply too_simple_server/database.py:27]: It's used further: `cursor.execute(f"create database {db_name}")`

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**